releases-implementation/README.md

```diff
@@ -0,0 +1,415 @@
+---
+creation_date: 2024-05-26
+issues:
+- TBA
+owners:
+- https://github.com/orgs/giantswarm/teams/sig-architecture
+- https://github.com/orgs/giantswarm/teams/team-turtles
+state: approved
+summary: Where and how we implement releases for workload clusters.
+---
+
+# Releases implementation
+
+## 1. Introduction
+
+This RFC defines how we implement releases for workload clusters for the new KaaS product that is based on the Cluster API project. It covers multiple aspects of releases, such as creation, testing and delivery of releases. It does not cover how clusters are upgraded to the new releases.
+
+## 2. Requirements language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119] [RFC8174].
```

Comment:
I guess these are external RFCs cited here, most probably will need linking

Reply:
They are (https://github.com/giantswarm/rfc/pull/96/files#diff-203a244ac080f7a2c2046fe64c789ea1a5b963578abcaba96c551f4db232b676R414-R415)

View the "rich diff" and you'll see the links working.